herddb-core/src/main/java/herddb/file/FileRecordSet.java

```diff
@@ -88,7 +87,11 @@ public void write(DataAccessor object, ExtendedDataOutputStream oo) throws IOExc
         if (!writeFinished) {
             throw new IllegalStateException("RecordSet is still in write mode");
         }
-        return buffer.iterator();
+        try {
+            return buffer.iterator();
```

Comment:
Why this try and rethrow without any other actions?

Reply:
I added debug at this point, I will remove the try.